Project4/notes-app/style.css

```diff
@@ -18,3 +18,30 @@ body {
   height: 400px;
   width: 400px;
 }
+.text-container{
+  position: relative;;
+}
+
+.text-container .cover{
+  padding: 0px 10px;
+  position: absolute;
+  left: 0;
+  top: 0;
+  background-color: rgb(162, 207, 103);
+  width: 300px;
```

Comment:
@zekeriyakoca why we added this width?

Reply:
It's a temporary solution. Please ignore for now